lib/nimble_template/addons/variants/phoenix/web/wallaby.ex

```diff
@@ -42,10 +42,13 @@ defmodule NimbleTemplate.Addons.Phoenix.Web.Wallaby do
   end
 
   defp inject_mix_dependency(%Project{} = project) do
-    Generator.inject_mix_dependency([
+    Generator.inject_mix_dependency(
       {:wallaby, latest_package_version(:wallaby), only: :test, runtime: false}
-    ])
+    )
 
+    # There is a conflict on `mime` version between Plug and Tesla.
+    # The Wallaby team hasn't used a the new Tesla version 1.4.4.
+    # TODO: Remove the mime update once Wallaby team uses the new Tesla version.
```

Comment:
Since Wallaby use `web_driver_version` as dependency
https://github.com/elixir-wallaby/web_driver_client/blob/master/mix.exs#L42
> {:web_driver_client, "~> 0.2.0"},

and `web_driver_version` use `testla`
https://github.com/elixir-wallaby/web_driver_client/blob/8dc46a325f90a120cfa3c65c19da68faef553299/mix.exs#L42
> {:tesla, "~> 1.3"},

so I think testla 1.4 should match right? 🤔

Reply:
Oh good catch @longnd let me check that

Reply:
yeah it works @longnd I have removed that in https://github.com/nimblehq/elixir-templates/pull/141/commits/b5b338ac3c07869d7bc0aaef25550517cce6611b